### ISSUE TITLE
TestTemporaryDirectory.test_flags fails on FreeBSD/ZFS

### ISSUE THREAD

**Issue by 8e73f1ab-f5c1-4ba5-8fa3-bf8b8561e287** [python/cpython#83228]
BPO | [39047](https://bugs.python.org/issue39047)
--- | :---
Nosy | @attilajeges
PRs | <li>python/cpython#17607</li>

<sup>*Note: these values reflect the state of the issue at the time it was migrated and might not reflect the current state.*</sup>

<details><summary>Show more details</summary><p>

GitHub fields:
```python
assignee = None
closed_at = None
created_at = <Date 2019-12-14.17:10:08.242>
labels = ['type-bug', 'tests', '3.9']
title = 'TestTemporaryDirectory.test_flags fails on FreeBSD/ZFS'
updated_at = <Date 2019-12-14.17:14:29.161>
user = 'https://github.com/attilajeges'
```

bugs.python.org fields:
```python
activity = <Date 2019-12-14.17:14:29.161>
actor = 'attilajeges'
assignee = 'none'
closed = False
closed_date = None
closer = None
components = ['Tests']
creation = <Date 2019-12-14.17:10:08.242>
creator = 'attilajeges'
dependencies = []
files = []
hgrepos = []
issue_num = 39047
keywords = ['patch']
message_count = 1.0
messages = ['358398']
nosy_count = 1.0
nosy_names = ['attilajeges']
pr_nums = ['17607']
priority = 'normal'
resolution = None
stage = 'patch review'
status = 'open'
superseder = None
type = 'behavior'
url = 'https://bugs.python.org/issue39047'
versions = ['Python 3.9']
```

</p></details>

**Comment by emaste** [comment]
I presume we would see the same behaviour on Linux with /tmp on ZFS.

**Comment by serhiy-storchaka** [comment]
It was fixed in a duplicate issue #108962.
